tests/optimization_test.py

```diff
@@ -46,5 +49,23 @@ def test_adam(self):
         self.assertListAlmostEqual(w.tolist(), [0.4, 0.2, -0.5], tol=1e-2)
 
 
+class WarmupCosineWithRestartsTest(unittest.TestCase):
+    def test_it(self):
+        m = WarmupCosineWithWarmupRestartsSchedule(warmup=0.05, t_total=1000., cycles=5)
+        x = np.arange(0, 1000)
+        y = [m.get_lr(xe) for xe in x]
+        # plt.plot(y)
+        # plt.show(block=False)
```

Comment:
If we no longer want to do this rendering in this test, we should remove this code from the test IMO.